code/framework/src/integrations/server/scripting/module.h

```diff
@@ -72,7 +72,9 @@ namespace Framework::Integrations::Server::Scripting {
 
       private:
         void UpdateFileWatcher();
-        void ReloadScriptingEngine();
         void SetupWatchPath(const std::string &path);
+        
+      public:
```

Comment:
Why did you put it at the end of the file? Move the method amongst other public methods.